src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -777,6 +778,10 @@ well as menu structures (for main menu and popup menus).
          <shortcut refid="openSourceDoc" value="Ctrl+O" disableModes="emacs"/>
          <shortcut refid="openSourceDoc" value="Meta+O" if="org.rstudio.core.client.BrowseCap.isMacintosh()"/>
 
+         <shortcut refid="openSourceDocNewColumn" value="Ctrl+X Ctrl+C" disableModes="default,vim,sublime"/>
+         <shortcut refid="openSourceDocNewColumn" value="Ctrl+Shift+O" disableModes="emacs"/>
+         <shortcut refid="openSourceDocNewColumn" value="Meta+Shift+O" if="org.rstudio.core.client.BrowseCap.isMacintosh()"/>
```

Comment:
On Mac (maybe others, didn't check) conflicts with toggleDocumentOutline shortcut.

Reply:
Thanks! I've changed it.